gateway-service-definitions/src/main/resources/services/rangerui/1.0.0/service.xml

```diff
@@ -32,5 +32,14 @@
             <rewrite apply="RANGERUI/rangerui/outbound/headers" to="response.headers"/>
         </route>
     </routes>
-    <dispatch classname="org.apache.knox.gateway.dispatch.DefaultDispatch"/>
+    <dispatch classname="org.apache.knox.gateway.dispatch.ConfigurableDispatch" ha-classname="org.apache.knox.gateway.dispatch.ConfigurableDispatch">
+        <param>
+            <name>requestExcludeHeaders</name>
```

Comment:
Not sure this needs to be specified? It should default if not specified?

Reply:
Right, this list is default and can be removed. I'll update the PR.